delta/json-schema/json-schema.adoc

```diff
@@ -0,0 +1,32 @@
+[[json-schema-for-deltas]]
+== JSON Schema for Deltas
+
+This folder contains:
+
+1. An [Essential](https://essential.metadev.pro) format 
+representation for Delta Protocol Structures [delta-protocol.ess](./delta-protocol.ess). 
+This file was created manually converting from the 
+[C# implementation](https://github.com/LionWeb-io/lionweb-csharp/blob/dsl-serialization-chunk/src/LionWeb-CSharp/core/serialization/Serialization.cs).
+2. A JSON Schema serialization from (1) derived automatically by Essential applying a 
+transformation. [derived/delta.json.schema](./derived/delta.json.schema) It can used for OpenAPI specs, f.e.
+
+=== Validation
+Sample online validator for JSON Schema: <https://json-schema.hyperjump.io/>
+
+=== Discussion
+
+Translation considerations from C# to Essential:
+
+1. The tranlation removed some type aliases to types used in C# and interfaces.
+2. The inheritance structure is still preserved in this version.
+
+
+Design considerations
+
+1. Inheritance and interfaces are convenient on C# but not really needed 
+on JSON. We should consider pros & cons about preserving it or *flattening* 
+the inheritance to simple structures.
+2. Non public types and intermediate types & abstractions can be avoided 
+from the final JSON Schema to make it as simple as possible.
+3. There are many concepts with some kind of variant permutations. 
```

Comment:
Agree: some kind of usage of traits/mixins might be useful to cut down on the *apparent* complexity due to the sheer number of variations.